### DISCUSSION TITLE
not close automatic Connections on Server

### DISCUSSION THREAD

**Discussion by PedramKamrani** [rabbitmq/rabbitmq-server#10626] [Other] [answered]
### Describe the bug

Hi guys I have rabbit in server in connection tab up to 800 have Connection open and  since correct  Restart Application on windows service go to connection open to 20  arrive to number  i want fix problem help please me thanks.

### Reproduction steps

.

### Expected behavior

.

**Comment by michaelklishin** [comment, marked as answer]
I'm afraid I do not understand the question.

Applications are supposed to close their connections when they are no longer needed (but in general, [use longer-lived connections](https://www.rabbitmq.com/docs/connections#lifecycle)), and [perform connection recovery](https://www.rabbitmq.com/docs/api-guide#recovery) when network connections fail. Most client libraries either provide an opinionated automatic recovery feature, or (Go client, Pika) document how to perform connection recovery manually.
